### PR TITLE
187 improve chesster notifications

### LINKED ISSUE #187: Improve forfeit win notification
Make a note that you can still choose to play the game if you want. A lot of people seem to have the idea that forfeit results (in case of no-show etc) are final which is not the case.

### PR DESCRIPTION
**Pull request by hopffgam**
- fixes #187 mention that players are allowed to reschedule their game
after a no-show
- Specifies that a withdrawal is effective for the full season as the
previous reply has lead to confusion by some players.

### CODE REVIEW

**Review by mathgeek — APPROVED** [review]

File: heltour/tournament/notify.py [hunk 1]
```diff
@@ -832,6 +832,7 @@ def notify_noshow_claim(round_, player, punishment, allow_continue, **kwargs):
         continue_url = abs_url(reverse('by_league:by_season:modrequest',
                                        args=[league.tag, season.tag, 'request_continuation']))
         message += '\nOtherwise, if you want to continue playing next round, <%s|click here>.' % continue_url
+    message += '\nNote that you can reach out to your opponent and try to reschedule the game.'
```
Comment (lakinwecker) [inline heltour/tournament/notify.py:835]: Is this true across all leagues?

Comment (hopffgam) [reply heltour/tournament/notify.py:835]: Yes, this statement is true across all leagues.

File: heltour/tournament/automod.py [hunk 2]
```diff
@@ -201,7 +201,8 @@ def claim_win_noshow_created(instance, **kwargs):
     if p.get_player_presence(instance.requester).online_for_game \
         and not p.get_player_presence(opponent).online_for_game \
         and timezone.now() > p.scheduled_time + timedelta(minutes=21):
-        instance.approve(response='You\'ve been given a win by forfeit.')
+        instance.approve(response='You\'ve been given a win by forfeit. \
+        It is still possible to reschedule and play the game if you want to.')
```
[the remark this answers is not in the document]
Comment (hopffgam) [reply heltour/tournament/automod.py]: I could not get that dedent function work for me. I am now using a single line.

File: heltour/tournament/automod.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 from django.db.models.signals import post_save
 from django.dispatch.dispatcher import receiver
 from heltour.tournament.tasks import pairings_published
+from textwrap import dedent
```
Comment (lakinwecker) [inline heltour/tournament/automod.py:6]: This import is no longer used.

**Review by lakinwecker — APPROVED** [review]

**Pull request merged**